routes/login.mjs

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+import passport from 'passport';
+
+const router = express.Router();
+
+router.post('/google', passport.authenticate('google-id-token'), (req, res) => {
```

Comment:
Maybe this should be namespaced, since we already have multiple providers (eg `/auth/google`)

Reply:
it is scoped. It's route is /login/google. The router is prefixed in routes/index.mjs